man/discr_si.Rd

```diff
@@ -8,7 +8,7 @@ Distribution}
 discr_si(k, mu, sigma)
 }
 \arguments{
-\item{k}{Positive integer, or vector of positive ingerers for which the 
+\item{k}{Positive integer, or vector of positive interers for which the 
```

Comment:
```suggestion
\item{k}{Positive integer, or vector of positive integers for which the 
```

Reply:
Note: this must change in the source code as that's where the documentation is derived from.

Reply:
I had to remove a trailing `}` from one of the data documentation files that was preventing roxygen2 from parsing things correctly, so I think this might be the reason why only this Rd file changed, but not the source file.